test/FakeStateMachine-run.ts

```diff
@@ -123,4 +124,60 @@ describe('FakeStateMachine#run()', () => {
       );
     });
   });
+
+  describe('when Pass state has undefined ResultPath', () => {
+    test('should not throw an Error', async () => {
+      const definition = require('./fixtures/definitions/pass-state-undefined-resultpath.json');
+      const fakeStateMachine = new FakeStateMachine(definition, {});
+
+      expect(
+        await fakeStateMachine.run({
+          changed: 'TBD',
+          unchanged: 'unchanged',
+        })
+      ).toEqual(
+        new RunStateResult(
+          {
+            changed: 'changed by pass 0',
+            unchanged: 'unchanged',
+          },
+          'Succeed',
+          null,
+          true
+        )
+      );
+    });
+  });
+
+  describe('should have execution path', () => {
+    const definition = require('./fixtures/definitions/execution-path-multiple.json');
+
+    test('execution path should be valid', async () => {
```

Comment:
"valid" is ambiguous, so could you use more detailed description? For example, "executionPath has names in executed states" (I have no confidence about english)

Reply:
No problem, I'll see if I can make it clearer.